app/api/io/order.rb

```diff
@@ -38,6 +38,8 @@ def self.json_field_for(attribute)
                                            user <=  user
 
                     autodetect_studies_projects <= autodetect_studies_projects
```

Comment:
can autodetect_studies_project go if you have the split options? doesn't look like it's used much elsewhere (couple places in Limber where set to true)

Reply:
Good suggestion, have implemented this now.